ReactWindows/ReactNative/Common/AccessibilityHelper.cs

```diff
@@ -0,0 +1,258 @@
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Automation.Peers;
+
+namespace ReactNative.Common
+{
+    /// <summary>
+    /// Accessibility helper methods.
+    /// </summary>
+    public static class AccessibilityHelper
+    {
+        #region ImportantForAccessibility - Provides support for ImportantForAccessibility property implementation.
+
+        /// <summary>
+        /// Initializes the value of AccessibilityView for <paramref name="uiElement"/> and its children,
+        /// based on the ancestor <paramref name="parentUIElement"/> settings. Must be callded
+        /// when an element is initially added to the visula tree.
```

Comment:
>visula [](start = 54, length = 6)

typo

Reply:
Fixed.

---
In reply to: [164941970](https://github.com/Microsoft/react-native-windows/pull/1595#discussion_r164941970) [](ancestors = 164941970)